homeassistant/components/co2signal/strings.json

```diff
@@ -6,7 +6,7 @@
           "location": "Get data for",
           "api_key": "[%key:common::config_flow::data::access_token%]"
         },
-        "description": "Visit https://co2signal.com/ to request a token."
+        "description": "Visit https://electricitymaps.com/free-tier to request a token."
```

Comment:
Side note: In a future PR it would be good to move the link to a description placeholder so it's not included in the text to translate.